core-bundle/src/Migration/Version413/RelLightboxMigration.php

```diff
@@ -82,16 +86,22 @@ private function getTargets(): array
     {
         $this->framework->initialize();
 
-        $schemaManager = $this->connection->createSchemaManager();
         $targets = [];
+        $processed = [];
+
+        /** @var array<SplFileInfo> $files */
+        $files = $this->resourceFinder->findIn('dca')->depth(0)->files()->name('*.php');
 
-        foreach ($schemaManager->listTableNames() as $tableName) {
-            try {
-                Controller::loadDataContainer($tableName);
-            } catch (\Throwable $e) {
+        foreach ($files as $file) {
+            if (\in_array($file->getBasename(), $processed, true)) {
                 continue;
             }
 
+            $processed[] = $file->getBasename();
+
+            $tableName = $file->getBasename('.php');
+            Controller::loadDataContainer($tableName);
+
```

Comment:
I think we should use the same basename here.
```suggestion
        foreach ($files as $file) {
            $tableName = $file->getBasename('.php');

            if (\in_array($tableName, $processed, true)) {
                continue;
            }

            $processed[] = $tableName;

            Controller::loadDataContainer($tableName);

```